i18n/pt.po

```diff
@@ -297,6 +297,10 @@ msgstr "Mapeamento do Google Band com Data Element.q"
 msgid "Click to show feedback"
 msgstr "Clique para mostrar comentários"
 
+#, fuzzy
+msgid "Imports"
+msgstr "Importar"
```

Comment:
We should avoid automatic translations (`#, fuzzy`) since most of the time they're not correct. Let's remove it.